misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/exception/ConverterException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.converters.exception;
+
+/**
+ * JSON converter exception.
+ *
+ * @since 2.0.0
+ */
+public class ConverterException extends Exception {
```

Comment:
Isn't the name too generic? What if we rename to `JsonToRecord...`?

Reply:
refactored in eced62e